src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -2109,6 +2157,23 @@ impl<T: AstInfo> AstDisplay for Explainee<T> {
 }
 impl_display_t!(Explainee);
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum ExplainFormat {
```

Comment:
nit: once we merge #13296 `ExplainFormat` and `ExplainConfig` will move into `mz-repr` along with the new `Explain` trait. 

We will need some glue code to instantiate `ExplainConfig` from a vector of parsed identifiers, but I can do that as part of the #13296 PR.